api/task.go

```diff
@@ -95,6 +101,27 @@ type TaskDatabaseSchemaUpdatePayload struct {
 	VCSPushEvent  *vcs.PushEvent   `json:"pushEvent,omitempty"`
 }
 
+// TaskDatabaseSchemaUpdateGhostSyncPayload is the task payload for gh-ost syncing ghost table.
+type TaskDatabaseSchemaUpdateGhostSyncPayload struct {
+	Statement     string         `json:"statement,omitempty"`
+	SchemaVersion string         `json:"schemaVersion,omitempty"`
+	VCSPushEvent  *vcs.PushEvent `json:"pushEvent,omitempty"`
+	// SocketFileName is the socket file that gh-ost listens on.
+	// The name follows this template,
+	// `./tmp/gh-ost.{{ISSUE_ID}}.{{TASK_ID}}.{{DATABASE_ID}}.{{DATABASE_NAME}}.{{TABLE_NAME}}.{{TIMESTAMP}}.sock`
+	SocketFileName string
+}
+
+// TaskDatabaseSchemaUpdateGhostCutoverPayload is the task payload for gh-ost switching the original table and the ghost table.
+type TaskDatabaseSchemaUpdateGhostCutoverPayload struct {
+}
+
+// TaskDatabaseSchemaUpdateGhostDropOriginalTablePayload is the task type for dropping the original table
+type TaskDatabaseSchemaUpdateGhostDropOriginalTablePayload struct {
+	DatabaseName string `json:"databaseName,omitempty"`
```

Comment:
Does this point to the original name like "foo" or the name with gh-ost postfix ?

Reply:
The name with gh-ost suffix, like `_tablename_del`.

Reply:
let's add a comment.

Reply:
Done.